docs: update jest.mock for 'NativeAnimatedHelper' in React Navigation… (#712)
callstack/react-native-testing-library: website/docs/ReactNavigation.md

import AppNavigator from '../AppNavigator';

// Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420
// Use with React Native <= 0.63
jest.mock('react-native/Libraries/Animated/src/NativeAnimatedHelper');

> // Use the following mock instead if you're running React Native@0.64.0 or higher

Reviewer: ```suggestion
// Use this instead with React Native >= 0.64
```

> // Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420

Reviewer: ```suggestion
// Silence the warning https://github.com/facebook/react-native/issues/11094#issuecomment-263240420
// Use with React Native <= 0.63
```
